src/ripple/rpc/impl/RPCHelpers.h

```diff
@@ -207,7 +207,10 @@ std::optional<Seed>
 parseRippleLibSeed(Json::Value const& params);
 
 std::pair<PublicKey, SecretKey>
-keypairForSignature(Json::Value const& params, Json::Value& error);
+keypairForSignature(
+    Json::Value const& params,
+    Json::Value& error,
+    std::optional<std::reference_wrapper<JsonContext>> context = std::nullopt);
```

Comment:
Consider changing to `std::optional<uint> const& apiVersion`. Actually, maybe we can even do this: `uint apiVersion = RPC::apiVersionIfUnspecified`?